doc/source/whatsnew/v0.21.0.txt

```diff
@@ -106,6 +106,8 @@ Reshaping
 
 Numeric
 ^^^^^^^
+- DataFrame.interpolate was not respecting limit_direction when using the default limit=None (unlimited). Specifically, it would always use limit_direction='forward' even when limit_direction was set otherwise. Now default limit=None will work with other directions. :issue:`16282`
```

Comment:
Thanks @TomAugspurger, I thought Jeff wanted me to somehow move the change to the 0.20 branch.  That change is in.  I will ping when green.